dev/cnf/gradle/scripts/fat.gradle

```diff
@@ -296,15 +296,39 @@ task runfat(type: Exec) {
   }
 }
 
+def buildFatEnabled = true; // TODO set this to false once build machines have been updated
+gradle.startParameter.taskNames.each {
+	if(it.contains("fat")) {
+		buildFatEnabled = true;
+	}
+}
+
 compileJava {
   dependsOn cleanFat
 }
 
 assemble {
```

Comment:
Is there a way we could use a different target instead of 'assemble' to keep the targets from doing different actions depending on properties?